README.md

```diff
@@ -124,40 +166,12 @@ Upload the icons to our Google Cloud Bucket:
 
 Add icon name to the list of possible icon values:
 
-1. `src/atoms/Icon/constants.js`
-
+1. Open `src/atoms/Icon/constants.js`
+2. Add the name of your icon in alphabetical order
 
 ## Contributing
 
-### Gotchas:
-
-#### SCSS / CSS and How our CSS-Module config works
-
-- Our CSS Module config creates complex, hashed classes like this: `animate-module__shake--3Q0r1` which is `[name of the component]__[local classname]--[hash:base64:5]'`.
-- When webpack comes across an imported SCSS (`import styles from './button.module.scss'`), a new class is created name-spaced with the directory from where that file exists and added to the file css output file.
-- That `import styles..` will return a "styles" object of key / value pairs in this format: `{ "plain classname": "complex hashed classname" }`
-- You can then add classes to your JSX thus: `className: styles['classname']`
-
-**To prevent a bloated final CSS file, DON'T use the `@import` to import scss files that contain CSS selectors. `%placeholders`, `@mixin`, `@function`s are OK.**
-
-
-### Coding styles and standards
-
-Most of our styles are enforced by linters ([stylelint](https://github.com/stylelint/stylelint), [eslint](http://eslint.org/), [eslint-plugin-markdown](https://github.com/jmcolella/eslint-plugin-markdown)), which must follow for the builds to pass.
-
-For anything not specifically addressed in the linters, use Airbnb's styleguides:
-
-- [JS](https://github.com/airbnb/javascript)
-- [React/JSX](https://github.com/airbnb/javascript/tree/master/react)
-- [CSS](https://github.com/airbnb/css)
-
-### Continuous Integration and Deployment
-
-We use [Buildkite](https://buildkite.com/policygenius) as a CI server. Every commit triggers a build with Buildkite that will run a series of checks and tests before giving a green light to merge to master.
-
-There is 1 pipelines associated with this project:
-
-[athenaeum pipeline](https://buildkite.com/policygenius/athenaeum) - builds the project, runs test. When a branch is then merged to master, a new build is triggered. After a build on master passes all of its, it will need to be manually unblocked before being deployed to production and published to npm. This manual unblock is a prompt requiring you to bump the package version, for which we use [Semantic Versioning](http://semver.org/).
+For more information on Contributing and issues, please view the [`athenaeum` Github wiki](https://github.com/policygenius/athenaeum/wiki).
 
 ### Code reviews
```

Comment:
Is this section still necessary?

Reply:
yeah seemed pretty self explanatory, I removed